### PR TITLE
Crud 5

### PR DESCRIPTION
**Pull request by Dabble362**
highlight the winning squares when winner is declared @jduffey

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -49,3 +49,7 @@ ul {
 .game-info {
   margin-left: 20px;
 }
+
+.winning-squares {
```
Comment (jduffey) [inline src/index.css]: Very minor suggestion but I would name this in the singular, i.e. `winning-square`.

**Review by jduffey — APPROVED** [review]
Approved with a very minor suggestion.

File: src/index.css (hunk 1 above)
Comment (Dabble362) [reply src/index.css]: I like these suggestions.  I appreciate your attention to detail.  It helps me be the developer I would like to be! @jduffey

**Comment by Dabble362** [comment]
Changed all occurrences of winning-squares to winning-square (cuz it's one rendered square at a time), even in the CSS.

**Comment by jduffey** [comment]
Looks good! My approval is still active so feel free to merge when you're ready.

**Pull request merged**